pages/pipelines/build_retention.md

```diff
@@ -56,3 +56,24 @@ Then, under the Build Retention pane, you can select a Build Retention Period
 up to the maximum build retention period your plan offers.
 
 <%= image "builds.png", width: 1404/2, height: 1608/2, alt: "Screenshot of the 'Build' settings" %>
+
+## Configuring build export
+
+> 📘 Enterprise feature
+> Build export is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+If you require to keep builds longer than your [Buildkite plan](https://buildkite.com/pricing)
+allows you can setup build export using your private Amazon S3 bucket. Buildkite will export json
+representations of your builds to your private Amazon S3 bucket as they are removed per the build
+retention period for your [Buildkite plan](https://buildkite.com/pricing).
+
+Select organization settings
+
+<%= image "organization-settings.png", alt: "Screenshot of the organization settings button" %>
+
+From the Organization Settings menu, select Settings under the Pipelines heading
+
+<%= image "organization-pipeline-settings.png", alt: "Screenshot of the organization pipeline settings" %>
+
+Then, under the *Export Build Data to S3 pane*, you can input your private Amazon S3 bucket location and
+click *Enable Export*.
```

Comment:
I'm on a mission to avoid adding too many screenshots to the docs. We can still use them when they'll really help a user, but I think these could be removed in favor of providing a link to the settings page:

```suggestion
To enable build exports:

1. Navigate to your [organization's pipeline settings](https://buildkite.com/organizations/~/pipeline-settings).
1. In the _Export Build Data to S3_ section, enter the URL for the Amazon S3 bucket to use.
1. Select _Enable Export_.
```

What do you think?